tests/Unit/EvidencioAPITest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+use App\EvidencioAPI;
+use GuzzleHttp\Exception\ClientException;
+
+class EvidencioAPITest extends TestCase
+{
+
+    public function testOverview()
+    {
+        $overview = EvidencioAPI::overview();
+
+        $this->assertArrayHasKey("allModels",$overview);
```

Comment:
Assert messages are really helpful in describing what the assert should test for